telemetry/packages/public-app/components/levitationHeight.tsx

```diff
@@ -0,0 +1,206 @@
+'use client';
+
+import { useState } from 'react';
+import { useQuery } from 'react-query';
+import { LevitationHeightResponse } from '@hyped/telemetry-types';
+
+import {
+  Card,
+  TabList,
+  Tab,
+  ProgressBar,
+  Text,
+  Flex,
+  Button,
+  Metric,
+  BadgeDelta,
+  TabGroup,
+  Badge,
+} from '@tremor/react';
+
+interface products {
+  title: string;
+  value: number | null;
+  metric: string;
+  location: string;
+}
+
+const products = [
+  {
+    title: 'Levitation: Pod 1',
+    value: 0,
+    metric: 'mm',
+    location: 'A',
+  },
+  {
+    title: 'Levitation: Pod 2',
+    value: 0,
+    metric: 'mm',
+    location: 'A',
+  },
+  {
+    title: 'Levitation: Pod 3',
+    value: 0,
+    metric: 'mm',
+    location: 'A',
+  },
+  {
+    title: 'Levitation: Pod 4',
+    value: 0,
+    metric: 'mm',
+    location: 'B',
+  },
+];
+
+export default function LevitationHeight() {
+  const [selectedIndex, setSelectedIndex] = useState(0);
+  const selectedLocation = selectedIndex === 0 ? 'A' : 'B';
+  const [show, setShow] = useState(false);
+
+  const { data, error } = useQuery<LevitationHeightResponse>(
+    'levitation-height',
+    async () =>
+      await fetch(
+        `${process.env.NEXT_PUBLIC_TELEMETRY_SERVER}/pods/pod_1/public-data/levitation-height?start=0`,
+      ).then(res => res.json()),
+    {
+      refetchInterval: 1000,
+    },
+  );
+
+  if (!data) return error;
+  console.log('typeof :' + (Object.keys(data)[0] === 'statusCode'));
+  if (Object.keys(data)[0] === 'statusCode') {
+    const error = true;
+    console.log('The data fetch request resulted in error ' + error);
+  }
+
+  console.log('Levitation fetch request result ' + Object.keys(data));
+ 
+  return (
+    <Card decoration="top" decorationColor="red" className="h-[560px]">
+      <Flex alignItems="start">
+        <Text>Levitation Height</Text>
+        <BadgeDelta
+          deltaType={
+            data[Object.keys(data)[0]][0]?.value > 0
```

Comment:
maybe extract this logic out - have a variable called isPodLevitated and explain how you are determining it with a comment